tests/house_tests.py

```diff
@@ -126,43 +134,52 @@ def form_data_filter_tests():
     }
 
 
-@pytest.mark.django_db
-class TestMyHouseViews:
-    def test_house_view_function_200(self, db, generate_get_request):
-        HouseFactory().save()
-        house = House.objects.all()[0]
-        try:
-            render = house_view(generate_get_request, house.house_id)
-            assert render.status_code == 200, "Status code is not 200"
-        except Exception:
-            assert False, "Error in house_view function"
-
-    def test_mine_page_expenses_table_views_templates(self):
-        try:
-            get_template(MINE_EXPENSES_TITLE_ROUTE)
-        except TemplateDoesNotExist:
-            assert False, f"Template {MINE_EXPENSES_TITLE_ROUTE} does not exist"
-
-    def test_mine_page_house_table_views_templates(self):
-        try:
-            get_template(MINE_HOUSE_TABLE_ROUTE)
-        except TemplateDoesNotExist:
-            assert False, f"Template {MINE_HOUSE_TABLE_ROUTE} does not exist"
-
-    def test_mine_page_house_table_title_views_templates(self):
-        try:
-            get_template(MINE_HOUSE_TABLE_TITLE_ROUTE)
-        except TemplateDoesNotExist:
-            assert False, f"Template {MINE_HOUSE_TABLE_TITLE_ROUTE} does not exist"
-
-    def test_mine_page_mine_page_views_templates(self):
-        try:
-            get_template(MINE_MINE_PAGE_ROUTE)
-        except TemplateDoesNotExist:
-            assert False, f"Template {MINE_MINE_PAGE_ROUTE} does not exist"
-
-    def test_mine_sidebar_views_templates(self):
-        try:
-            get_template(MINE_SIDEBAR_ROUTE)
-        except TemplateDoesNotExist:
-            assert False, f"Template {MINE_SIDEBAR_ROUTE} does not exist"
+@pytest.fixture
```

Comment:
Why are these tests within this PR?
I don't see that you mentioned that the PR is blocked by another PR

Reply:
I branched from the main branch so I don't think this PR is blocked by any other PR. maybe I don't understand the question